### PR TITLE
feat(manager): upgrade DigitalOcean image

### PR DESCRIPTION
**Pull request by fortuna**
Upgrade to use image `docker-20-04`. Also enable the web console to make it easy to SSH into the machine.

### CODE REVIEW

File: src/server_manager/cloud/digitalocean_api.ts [hunk 1]
```diff
@@ -139,6 +140,9 @@ export class RestApiSession implements DigitalOceanSession {
           user_data: dropletSpec.installCommand,
           tags: dropletSpec.tags,
           ipv6: true,
+          // We install metrics and droplet agents in the user_data script in order to not delay the droplet readiness.
```
Comment (jyyi1) [inline src/server_manager/cloud/digitalocean_api.ts:143]: I'm curious what will the user observe? A improvement of the server creation time?

**Review by jyyi1 — APPROVED** [review]

File: src/server_manager/web_app/digitalocean_account.ts [hunk 2]
```diff
@@ -101,7 +101,7 @@ export class DigitalOceanAccount implements digitalocean.Account {
     const dropletSpec = {
       installCommand,
       size: MACHINE_SIZE,
-      image: 'docker-18-04',
+      image: 'docker-20-04',
```
Comment (daniellacosse) [inline src/server_manager/web_app/digitalocean_account.ts:104]: Where is this string sourced from? might be nice to comment a link here for future upgrades

**Review by daniellacosse — APPROVED** [review]

File: src/server_manager/web_app/digitalocean_account.ts (hunk 2 above)
Comment (fortuna) [reply src/server_manager/web_app/digitalocean_account.ts:105]: Good point. Added: https://slugs.do-api.dev/

File: src/server_manager/install_scripts/do_install_server.sh [hunk 3]
```diff
@@ -195,9 +195,15 @@ done
...
+# We could install the agents below in the create droplet request, but they add
+# over a minute of delay to the droplet readiness. Instead, we do it here.
 # Since the server manager looks only for the tags created in the previous
 # step, this does not slow down server creation.
+
+# Install the DigitalOcean Metrics Agent, for improved monitoring:
+# https://docs.digitalocean.com/products/monitoring/how-to/install-agent/
 curl -sSL https://repos.insights.digitalocean.com/install.sh | bash
+
+# Install the DigitalOcean Droplet Agennt, for web console integration:
```
[the remark this answers is not in the document]
Comment (fortuna) [reply src/server_manager/install_scripts/do_install_server.sh]: Fixed

File: src/server_manager/cloud/digitalocean_api.ts (hunk 1 above)
Comment (fortuna) [reply src/server_manager/cloud/digitalocean_api.ts:143]: The `monitoring: false` is a no-op, as that's the default value.
`with_droplet_agent: false` reduces the creation time, as the default installs that agent.

File: src/server_manager/electron_app/build.action.sh [hunk 4]
```diff
@@ -54,6 +54,9 @@ mkdir -p "${STATIC_DIR}/server_manager"
 cp -r "${OUT_DIR}/js/electron_app/"* "${STATIC_DIR}"
 cp -r "${BUILD_DIR}/server_manager/web_app/static" "${STATIC_DIR}/server_manager/web_app/"
 
+# TODO(fortuna): Separate the build of Electron main and the Electron package.
```
[the remark this answers is not in the document]
Comment (fortuna) [reply src/server_manager/electron_app/build.action.sh:57]: Would you ever start without building it first? In practice I feel like we will be writing build & start in the CLI all the time.

**Review by daniellacosse — APPROVED** [review]
Sweet!

File: src/server_manager/electron_app/build.action.sh (hunk 4 above)
[the remark this answers is not in the document]
Comment (jyyi1) [reply src/server_manager/electron_app/build.action.sh:57]: In some debugging cases such as debugging the start up behavior, I will run it multiple times and set different breakpoints.

**Pull request merged**